src/views/SalesInfo/components/index.ts

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components';
+import { ResponsiveContainer } from 'recharts';
+import { GU } from 'components/theme';
+import { media } from 'components/breakpoints';
+
+export const StyledButtonContainer = styled.div`
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: ${8 * GU}px ${10 * GU}px 0;
+
+  div {
+    margin-bottom: ${8 * GU}px;
+    transition: all 0.2s ease;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(${-GU}px);
```

Comment:
Instead of `translateY`, let's use `translate3d`—It's [faster](https://developer.mozilla.org/en-US/docs/Learn/Performance/CSS) on most browsers due to GPU acceleration!

Reply:
Fixed in 623a3f3344bbba3fde659232ee0f44c7f0104781